stock_reserve_mrp/mrp.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# © 2015 ClearCorp S.A.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+
+
+class mrp_bom(models.Model):
```

Comment:
in stock.py you use camel case for the class name which is the recommended practise. Can you rename this class to MrpBom?